Airports/EGXW/Positions.txt

```diff
@@ -1,3 +1,3 @@
-EGXW_APP:Waddington Approach:119.500:XWR:R:EGXW:APP:-:-:3601:3634:N053.09.57.000:W000.31.20.000
+EGXW_APP:Waddington Approach:128.955:XWR:R:EGXW:APP:-:-:3601:3634:N053.09.57.000:W000.31.20.000
```

Comment:
The Approach freq you have changed to is new. if you ware removing the Zone freq altogether then it should be added as it's own position.

Reply:
Should change it to 119.505 instead and add a new 128.955?